addon/styles/addon.css

```diff
@@ -96,3 +96,30 @@
 .sf-files-card--small > div.au-u-margin-top-tiny {
   margin-top: 0 !important;
 }
+
+.center-container {
```

Comment:
Appuniversum has some [flex utils](https://appuniversum.github.io/ember-appuniversum/?path=/story/utilities-flex--page), so you could use those if you want.